src/components/CategoriesNav/style.css

```
@@ -10,10 +10,15 @@
   transition: opacity 0.1s;
 }
 
+.emoji-picker-react .emoji-categories button:focus-visible {
+  background-color: #99c2f160;
```

Looks like you added an extra `60` at the end

```suggestion
  background-color: #99c2f1;
```